src/app/api/github-webhook/route.ts

```diff
@@ -0,0 +1,90 @@
+import { NextRequest, NextResponse } from 'next/server';
+import crypto from 'crypto-js';
+import db from '@/db';
+import {
+  formatINR,
+  formatUSD,
+  getCurrencyRate,
+  sendBountyComment,
+} from '@/utiles/bounty';
+
+export async function POST(req: NextRequest) {
```

Comment:
Why are we reinventing the wheel???

Reply:
This code is already done in bounty hook right?

Reply:
Yes, We are getting this information from bounty-hook only and not from the github-webhook directly.
Also created PR  in the bounty-hook repo for the same.

Reply:
Flow : github-webhook > bounty-hook (checks for bounty msg) > cms (storing db) > github comment (if success)